float32 pack_voltage
float32[] cell_voltages
int32 charge
int32 status
